redis-persistence/src/main/java/com/netflix/conductor/redis/dynoqueue/RedisPinger.java

```diff
@@ -23,9 +25,53 @@
 
 public class RedisPinger {
 
-    private static final Logger log = LoggerFactory.getLogger(RedisPinger.class);
+    private static final Logger logger = LoggerFactory.getLogger(RedisPinger.class);
     private static final int CONNECTION_TIMEOUT = 3000;
 
+    private static int MAX_RETRY_COUNT = 3;
+
+    private Integer waitObject = Integer.valueOf(1);
+
+    /**
+     * Does a redis ping to the host. If pings returns true, this function returns true. If ping
+     * fails 2 more times ping will be done with an interval of 1 second. If fails in all retries
```

Comment:
From the function implementation, looks like we are doing exponential backoff and internal is no long 1 second. Can you please update the function comments to reflect the implementation.

Reply:
done